tests/playback-ui.js

```diff
@@ -1,18 +1,19 @@
+/* global VNC_frame_data, VNC_frame_encoding */
+
 import * as WebUtil from '../app/webutil.js';
 import RecordingPlayer from './playback.js';
 
-var frames = null;
-var encoding = null;
+let frames = null;
+let encoding = null;
 
 function message(str) {
-    console.log(str);
-    var cell = document.getElementById('messages');
+    const cell = document.getElementById('messages');
     cell.textContent += str + "\n";
     cell.scrollTop = cell.scrollHeight;
 }
 
 function loadFile() {
-    const fname = WebUtil.getQueryVar('data', null);
+    const fname = WebUtil.getQueryconst('data', null);
```

Comment:
Something went very wrong here. `getQueryconst` does not exist. Looks like some kind of manual search-and-replace for `var`.